.github/workflows/rust.yml

```diff
@@ -248,11 +248,11 @@ jobs:
           rustup toolchain install stable
           rustup default stable
           rustup component add rustfmt
-      - name: Run tests
+      - name: Run tests (excluding doctests)
         shell: bash
         run: |
           export PATH=$PATH:$HOME/d/protoc/bin
-          cargo test
+          cargo test --lib --tests --bins --features avro,jit,scheduler,json,dictionary_expressions
```

Comment:
I think it is a reasonable idea to run with the same features on mac and windows. Thank you @jiangzhx